DSharpPlus/Entities/Channel/Message/DiscordMessageBuilder.cs

```diff
@@ -112,7 +112,7 @@ public DiscordMessageBuilder WithContent(string content)
         /// <returns>The current builder to be chained.</returns>
         /// <exception cref="ArgumentOutOfRangeException">No components were passed.</exception>
         public DiscordMessageBuilder AddComponents(params DiscordComponent[] components)
-            => this.AddComponents((IEnumerable<DiscordComponent>)components);
+            => this.AddComponents(components);
```

Comment:
This is recursive now, and will cause a stack overflow, since this is the only overload that takes an array.